app/controllers/TransferCompleteController.scala

```diff
@@ -1,16 +1,16 @@
 package controllers
 
-import java.util.UUID
-
 import auth.TokenSecurity
 import configuration.KeycloakConfiguration
-import javax.inject.Inject
 import org.pac4j.play.scala.SecurityComponents
 import play.api.i18n.I18nSupport
-import play.api.mvc.{Action, AnyContent, Request, RequestHeader}
+import play.api.mvc.{Action, AnyContent, Request}
 import services.ConsignmentService
 
-import scala.concurrent.{ExecutionContext, Future}
+import java.time.{LocalDateTime, ZonedDateTime}
+import java.util.UUID
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
 
 class TransferCompleteController @Inject()(val controllerComponents: SecurityComponents,
```

Comment:
I'd add additional test to check that the new HTML for the download report is appearing. See line 50

Reply:
ah yes I forgot about those one. I've added a test for that now